apps/console/src/features/console-settings/components/console-administrators/administrators-list/administrators-table.tsx

```diff
@@ -328,16 +331,16 @@ const AdministratorsTable: React.FunctionComponent<AdministratorsTablePropsInter
         const actions: TableActionsInterface[] = [
             {
                 "data-componentid": "administrators-list-item-edit-button",
-                hidden: (): boolean => !isFeatureEnabled(featureConfig?.users,
+                hidden: (): boolean => !isFeatureEnabled(featureConfig,
                     UserManagementConstants.FEATURE_DICTIONARY.get("USER_READ")),
                 icon: (user: UserBasicInterface): SemanticICONS => {
                     const userStore: string = user?.userName?.split("/").length > 1
                         ? user?.userName?.split("/")[0]
                         : "PRIMARY";
 
                     return (
-                        !hasRequiredScopes(featureConfig?.users, featureConfig?.users?.scopes?.update, allowedScopes)
-                    || !isFeatureEnabled(featureConfig?.users,
+                        !hasRequiredScopes(featureConfig, featureConfig?.scopes?.update, allowedScopes)
```

Comment:
is `featureConfig?.scopes?.update` intentional?

PS: Later only I saw that `featureConfig` refers to the configs for users feature. In other places, we assign the whole feature config to this variable and use feature specific configs as `featureConfig.featureName`. Better to stick to that approach to maintain consistency.

Reply:
As explained in this guide https://github.com/wso2/identity-apps/blob/master/docs/write-code/PERFORMANCE.md#dont-write-redundant-code we should reduce the unnecessary usage of optional chaning to reduce the bundle size. If we use the complete featureConfig we have to add optional chaning each and every place that we are using it. So, we need to update the other places to strict with this method.